templates/spiders/csvfeed.tmpl

```diff
@@ -14,16 +17,15 @@ class $classname(CSVFeedSpider):
         # "brand_wikidata": "$brand_wikidata",
         # "operator": "$operator",
         # "operator_wikidata": "$operator_wikidata",
-        # "extras": Categories.SHOP_XYZ.value
     }
 
     # Do any adaptations you need here
     #def adapt_response(self, response):
     #    return response
 
     def parse_row(self, response, row):
-        i = {}
-        #i["url"] = row["url"]
-        #i["name"] = row["name"]
-        #i["description"] = row["description"]
+        i = Feature()
+        # i["website"] = row["url"]
+        # i["name"] = row["name"]
```

Comment:
Suggest changing this default example to `ref` rather than `name` because `ref` is more-or-less mandatory, but `name` isn't.